.dockerignore

```diff
@@ -4,7 +4,7 @@
 dist
 node_modules
 
-config/config.example.json
+config/config.json
```

Comment:
Shouldn't this line be removed? Then it'd be still correct to tell the user to create a `config.json` in that config folder, it would get copied to the Docker container anyway.

Reply:
Configuration/environment-specific information generally shouldn't be built into a Docker image.  I suppose a default config.json file could be committed and built into the image if that's what you would prefer, but the end user would still want to mount their own config as a volume.